docs/standard_report.md

```diff
@@ -143,31 +146,32 @@ In this report, we will find out how many applicants received full/provisional l
 **Developer Side:** \
 It is pretty straightforward. We need to identify the highest milestone, full or provisional between `from` and `to` date.
 Now we have to show it in a separate row.\
-For that, we will use UNION all again with a static filter on milestone one query for full, and one query for provisional.
+For that, we will use UNION all again with a static filter on milestone two queries(LPN/ RN) for full and two queries(LPN/ RN) for provisional.
 
 
 
 
 ***
 ### Report 6: Number of Internationally Educated Nurse Registrants in the Recruitment Stage
 
-Applicants can apply for multiple job-competition under the same health authorities or different health authorities.\
-In this report, if an applicant has applied in more than one job competition then that applicant count must be shown multiple times in different HA-related job competitions in the report.
+Applicants can apply for multiple job-competitions under the same health authorities or different health authorities.\
```

Comment:
for consistency, why don't we choose one? `job-competition` or `job competition`

Reply:
will do, will give it another look over and fix it up